docs/sources/review/lint-prose/index.md

```diff
@@ -110,7 +125,11 @@ If you are installing Vale on macOS, make sure that [Homebrew](https://brew.sh/)
    TokenIgnores = (<http[^\n]+>+?), \*\*[^\n]+\*\*
    ```
 
-   Replace `FULL_PATH_TO_REPO` with the full path to the cloned Writer's Toolkit repository. For example, in Linux you could set StylesPath to `/home/username/git-repos/writers-toolkit/vale` and in macOS, you could set it to `/Users/username/git-repos/writers-toolkit/vale`. The path depends on where you cloned the git repository.
+   Replace `FULL_PATH_TO_REPO` with the full path to the cloned Writer's Toolkit repository. The path depends on where you cloned the git repository. For example:
+
+   - On Linux, you could set StylesPath to `/home/USERNAME/git-repos/writers-toolkit/vale`
+   - On macOS, you could set StylesPath to `/Users/USERNAME/git-repos/writers-toolkit/vale`
+   - On Windows, you could set StylesPath to `C:\Users\USERNAME\git-repos\writers-toolkit\vale`
```

Comment:
I believe our placeholders should follow the our style guidance and include angled brackets.
That being said, I think our documentation there needs revision so I'm going to open a PR to fix it.

https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#:~:text=Use%20descriptive%20words,following%20raw%20markdown%3A.

Similar changes should be applied to the use of `FULL_PATH_TO_REPO` throughout this page.

```suggestion
   - On Linux, you could set StylesPath to `/home/<USERNAME>/git-repos/writers-toolkit/vale`
   - On macOS, you could set StylesPath to `/Users/<USERNAME>/git-repos/writers-toolkit/vale`
   - On Windows, you could set StylesPath to `C:\Users\<USERNAME>\git-repos\writers-toolkit\vale`
```

Reply:
I used the placeholder text style still in use in the Agent repo. Updating here... Agent will be different for now...

Reply:
Yeah, I think the guidance is relatively new and it's not used consistently yet but we have to start somewhere :)